Formula/e/easy-rsa.rb

```diff
@@ -19,7 +19,12 @@ class EasyRsa < Formula
   depends_on "openssl@3"
 
   def install
-    inreplace "easyrsa3/easyrsa", "'/etc/easy-rsa'", "'#{pkgetc}'"
+    inreplace "easyrsa3/easyrsa" do |s|
+      s.gsub! "'/etc/easy-rsa'", "'#{pkgetc}'"
+      s.gsub! "~VER~", version.to_s
+      s.gsub! "~DATE~", time.iso8601.to_s
+      s.gsub! "~GITHEAD~", "RELEASE"
```

Comment:
Might be easier to use the release tarball at https://github.com/OpenVPN/easy-rsa/releases/download/v3.2.0/EasyRSA-3.2.0.tgz that includes these values, so we don't need to `inreplace` them.

Reply:
Huh, that's probably a better solution. I assuming switching sources like this warrants a revision bump?

Reply:
Probably not, TBH. We can revision bump it if someone else complains about it.